### PR TITLE
PublishHandler fix for iOS OwnTracks payload

### PR DESCRIPTION
**Pull request by aleohl**
Fixes issue #2

### CODE REVIEW

File: orion/handlers/publish_handler.py [hunk 1]
```diff
@@ -23,8 +23,13 @@ def run(self, *args, **kwargs):
         if self.data['_type'] != 'location':
             return self.error(status=400, message='Not a location publish.')
 
-        user = request.headers.get('X-Limit-U')
-        device = request.headers.get('X-Limit-D')
+        if self.data.get('topic'):
+            topic = self.data.get('topic').split('/')
```
Comment (LINKIWI) [inline orion/handlers/publish_handler.py]: To be more Pythonic:

`_, user, device = self.data.get('topic').split('/')`

**Comment by LINKIWI** [comment]
https://github.com/LINKIWI/orion-server/blob/master/test/handlers/test_publish_handler.py#L28 would be a good test case to copy, removing `mock_headers` and adding `topic` to the mock data payload.

**Comment by aleohl** [comment]
Sorry, not so experienced in python yet. Usually programs php. Will have a look at adding a test case and be more pythonic! Thanks for great reply.

**Comment by LINKIWI** [comment]
No worries, thanks for your contribution!

**Comment by aleohl** [comment]
Fixed as per your request. Happy to contribute!

**Comment by LINKIWI** [comment]
Thanks!

**Pull request merged**